[Help] upload.yaml中的Tag字段为空，导致上传后投稿失败

## Problem Overview
执行`/app/src/utils/biliup -u  /app/src/utils/cookies.json upload /app/Videos/1847879043/1847879043_2025-03-18-19-51-.mp4 --config /app/src/upload/upload.yaml`后返回：`Error: ResponseData { code: 21005, data: None, message: "Tag不能为空，总数量不能超过12个， 并且单个不能超过20个字", ttl: Some(1) }`

检查后发现upload.yaml中对应的视频下tag字段为空

## Steps Taken
1. 录制
2. 执行`/app/src/utils/biliup -u  /app/src/utils/cookies.json upload /app/Videos/1847879043/1847879043_2025-03-18-19-51-.mp4 --config /app/src/upload/upload.yaml`
3. See errors: `Error: ResponseData { code: 21005, data: None, message: "Tag不能为空，总数量不能超过12个， 并且单个不能超过20个字", ttl: Some(1) }`

## Expected Outcome
Tag字段内应该有至少一个以上的标签
手动添加了一个tag后投稿成功

## Screenshots

## Environment Information
 - Operating System: Ubuntu 24.04 Docker 28.0.1
 - Python Version: Python 3.10.16 (Latest docker image)
 - Driver & CUDA Version: none
 - Error Messages and Logs: `ResponseData { code: 21005, data: None, message: "Tag不能为空，总数量不能超过12个， 并且单个不能超过20个字", ttl: Some(1) }`

## Additional context
upload log
```
[INFO] - [2025-03-18 20:04:13,355 bilive upload] - deal with /app/Videos/1847879043/1847879043_2025-03-18-19-51-.mp4
[INFO] - [2025-03-18 20:04:13,985 bilive upload] - First upload this live
[ERROR] - [2025-03-18 20:06:26,781 bilive upload] - The upload_video called failed, the files will be reserved. error: Command '['/app/src/utils/biliup', '-u', '/app/src/utils/cookies.json', 'upload', '/app/Videos/1847879043/1847879043_2025-03-18-19-51-.mp4', '--config', '/app/src/upload/upload.yaml']' returned non-zero exit status 1.
```

--- a/src/upload/extract_video_info.py
+++ b/src/upload/extract_video_info.py
@@ -55,7 +55,12 @@ def generate_desc(video_path):
 def generate_tag(video_path):
     title, artist, date = get_video_info(video_path)
     artist_text = re.sub(r'\W+', '', artist)
+    
     tags = get_bilibili_suggestions(artist_text)
+    
+    if not tags:
+        tags = ["直播回放", "切片"]
+    
     return tags
 
 def generate_source(video_path):

